packages/embed/src/factories/create-popup/popup-options.ts

```diff
@@ -39,4 +39,16 @@ export type PopupOptions = UrlOptions &
      * @type {HTMLElement}
      */
     container?: HTMLElement
+    /**
+     * Specify custom launch options for Popup
+     *
+     * @type {string}
+     */
+    open?: string
```

Comment:
Maybe we could type this? [We define this in README](https://github.com/Typeform/embed#popup) as:
> String any of: "exit" "load"  "scroll" "time"

We could type this as:
```suggestion
    open?: 'exit' | 'load' | 'scroll' | 'time'
```